### PR TITLE
Implement tokens for unsigned int and double data types

### LINKED ISSUE #58: Add `Double` Data Type Token at translator frontend
Implement the DOUBLE token to recognize the double data type for representing double-precision floating-point numbers.

### LINKED ISSUE #59: Add `Unsigned Int ` Data Type Token at translator frontend
Implement the UINT token to recognize the uint data type for representing unsigned integers.

### PR DESCRIPTION
**Pull request by rigved-desai**
### PR Description
This PR introduces the following changes:
1. Tokens for ```unsigned int``` (```UINT```) and ```double``` (```DOUBLE```) have been implemented in ```lexer.py```.
2. A test has been added for data type tokenization in ```test_lexer.py```,

### Related Issue

This PR attempts to solve the following issues:
Closes #58
Closes #59

### Checklist
- [x] Have you added the necessary tests?
- [x] Only modified the files mentioned in the related issue(s)?
- [x] Are all tests passing?

### CODE REVIEW

File: crosstl/src/translator/lexer.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
     ("FLOAT_NUMBER", r"\d*\.\d+|\d+\.\d*"),
     ("FLOAT", r"\bfloat\b"),
     ("INT", r"\bint\b"),
+    ("UINT", r"\bunsigned\s+int\b"),
```
Comment (samthakur587) [inline crosstl/src/translator/lexer.py:20]: hii @rigved-desai for the UINT token we are defining `uint` not `unsigned int` can you please change it to `uint` only .

**Review by samthakur587 — CHANGES_REQUESTED** [review]
change `unsigned int` to `uint`

File: crosstl/src/translator/lexer.py (hunk 1 above)
Comment (rigved-desai) [reply crosstl/src/translator/lexer.py]: Updated in latest commit 👍

**Review by samthakur587 — APPROVED** [review]
great work @rigved-desai  everything LGTM 🚀

**Pull request merged**